roadmap/roadmap.md

```diff
@@ -36,52 +37,81 @@ This Roadmap captures high level actionable items that we as a project aim to un
+
+- :large_orange_diamond: Revive dashboards for developers (CI status etc) and community (PR trends etc).
+
 
-- :large_orange_diamond: Improve support and validation for input/output of standard file formats used in Astronomy by improving the unified I/O system.
+## Interoperability and compatibility
 
-- :large_orange_diamond: Improve interoperability and compatibility, while reducing duplication, amongst Astropy-affiliated packages and other community standard software.
+One of the core principles of the Astropy Project is to "foster an ecosystem of interoperable astronomy packages." This category encompasses goals related to that principle of interoperability and compatibility.
+
+- :green_circle: Improve support and validation for input/output of standard file formats used in Astronomy by improving the unified I/O system: allowing user added classes/formats, and implementing GWCS.
+
+- :red_square: Unify unit packages.
+    
 
 ## Hardware and Performance
 
+Astronomers perform their work in a variety of computing environments with a variety of requirements. This category encompasses goals related to supporting the performance and hardware needs of Astropy Project users, as well as  anticipating future needs and requirements.
+
 - :large_orange_diamond: Improve and/or maintain interoperability with performant I/O file formats and libraries such as HDF5 and Dask.
 
 - :red_square: Improve support for using Astropy tools in heterogeneous computing environments such as cloud environments or GPU systems.
 
+- :red_square: Add basic pyscript/webassembly support in Astropy.
+
+
 ## Learn and User Support
 
+The Astropy Project cannot succeed in its core objectives without a robust user onboarding and support system.  This category is for items related to the Astropy Learn ecosystem and other user support related improvements.
+
 - :green_circle: Generate and ingest guides and/or a series of tutorials that demonstrate Astropy Project functionality in the context of astronomical research.
 
--  :large_orange_diamond: Update Astropy website: Rework the Team page and improve the visibility of Astropy Learn.
+-  :large_orange_diamond: Update Astropy website to improve the visibility of Astropy Learn and overall reflect the Project as a whole, not just the core package.
 
-- :large_orange_diamond: Expand and build on workshop offerings by recruiting and training more facilitators and expanding the geographic diversity of meetings where we offer workshops. 
+- :large_orange_diamond: Develop tutorials that demonstrate the use of Astropy for spectroscopy tasks.
 
-- :large_orange_diamond: Expand the number and diversity of active user support resources.
+- :red_square: Expand and build on workshop offerings, including online and increased geographic coverage.
 
-- :red_square: Develop tutorials that demonstrate the use of Astropy for spectroscopy tasks.
+- :red_square: Develop/harvest tutorials suitable for use in university astronomy courses.

```

Comment:
Do we want it more vague to include non-traditional tertiary education that we are not aware of?
```suggestion
- :red_square: Develop/harvest tutorials suitable for use in post-secondary academic settings.
```

Reply:
I don't think so. I don't want to over-promise and this is not saying that these tutorials are *only* for university courses, just that we are building a collection that is *suitable* for university courses, and if they are suitable for other courses too, great.